src/main/kotlin/com/jfrog/conan/clion/dialogs/ConanExecutableDialog.kt

```diff
@@ -26,10 +26,6 @@ object ConanExecutableChooserDescriptor : FileChooserDescriptor(true, true, fals
         withFileFilter { it.isConanExecutable }
         withTitle("Select Conan executable")
     }
-
```

Comment:
I think we should also set [FileChooserDescriptor](https://github.com/JetBrains/intellij-community/blob/af3ce6ded80b111c45dc31d4034cdb9924dd08e6/platform/ide-core/src/com/intellij/openapi/fileChooser/FileChooserDescriptor.java#L32) to:
So it does not choose folders
```
FileChooserDescriptor(true, false, false, false, false, false) {
```